landing-page/components/Header.tsx

```diff
@@ -46,13 +50,32 @@ const items: SubLinkType[] = [
   },
 ]
 
+const defaultHeaderTranslation = {
```

Comment:
Type `HeaderTranslation` are missing entirely
```suggestion
const defaultHeaderTranslation: Record<Languages, HeaderTranslation> = {
```